recipe/patches/0001-BLD-Use-shared-library-deps.patch

```diff
@@ -1,5 +1,60 @@
+From 3ddc6337ab3f33944c61a208fdac3ce916293cdf Mon Sep 17 00:00:00 2001
+From: Daniel Ching <dching@nvidia.com>
+Date: Fri, 18 Apr 2025 10:57:08 -0500
+Subject: [PATCH 1/5] BLD: Use shared library deps
+
+---
+ cmake/Dependencies.aarch64-linux.cmake | 22 +++++-----------------
+ cmake/Dependencies.cmake               | 19 ++++++-------------
+ cmake/Dependencies.common.cmake        |  1 -
+ cmake/modules/FindNVJPEG.cmake         |  4 ++--
+ 4 files changed, 13 insertions(+), 33 deletions(-)
+
+diff --git a/cmake/Dependencies.aarch64-linux.cmake b/cmake/Dependencies.aarch64-linux.cmake
+index f93e1c07..ac010892 100644
+--- a/cmake/Dependencies.aarch64-linux.cmake
++++ b/cmake/Dependencies.aarch64-linux.cmake
```

Comment:
These is not the file you are looking for.
`cmake/Dependencies.aarch64-linux.cmake` is used only for `aarch64-linux` which we use for [Jetson when cross compiling](https://github.com/NVIDIA/DALI/blob/main/docker/Dockerfile.build.aarch64-linux#L125C9-L125C23). SBS uses just `aarch64`.
Also I think the recommendation is to link runtime statically for [the minor version compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/#minor-version-compatibility) - so we don't have to worry of using functions in DALI that are not available in the user CTK installation. cudart is relatively small and should not be an issue for the user to ship it inside the DALI binary. On the other hand this is conda where everything is inside the conda deployment so it may be not a bad idea after all.

Reply:
Or if you truly want to cross compile for SBSA maybe this file can be applied... For sure it was created with Jetson specifics in mind and I don't know how well it fits there.